docker-compose.yml

```diff
@@ -195,6 +195,10 @@ services:
         - ghcr.io/samvera/hyku/base:latest
         - ghcr.io/samvera/hyku/web:latest
         - ghcr.io/samvera/hyku/worker:latest
+    entrypoint: ["sh", "-c"]
+    command:
+    - >
+      echo 'running migrations'; ./bin/db-migrate-seed.sh
```

Comment:
Since we change the path we aren't able to run this in the web container. The deployment doesn't reference the db-migrate-seed.sh script so I think it's safe to pull out. Open to feedback.

Reply:
welp, i forgot about the db-setup init container on the web helm deploy :( So I do need to solve this in the web service. Taking this out.